docs/dataset-overviews/anthropogenic-slcf-co2-emissions.md

```diff
@@ -1,83 +1,176 @@
+- [CEDS-CMIP-2024-11-25](https://aims2.llnl.gov/search?project=input4MIPs&versionType=all&&activeFacets=%7B%22source_id%22%3A%22CEDS-CMIP-2024-11-25%22%7D).
+
+Retrieving and only using valid data will require some care.
+Please make sure you read the guidance given at the start of this Summary section
+and process the data carefully.
+
+This data is for testing (both of the forcing data and of modelling workflows) only.
+Production simulations should not be started based on any data that has a `mip_era` value equal to 'CMIP6Plus'.
+(The `mip_era` metadata value appears both in each file's global attributes as well as its metadata on ESGF.)
+
+If you have any feedback, please add it to the [relevant GitHub discussion](https://github.com/PCMDI/input4MIPs_CVs/discussions).
+
+#### CMIP7 AR7 fast track
+
+No data available for this phase yet.
+
+This data will be for the CMIP7 AR7 fast track.
+All data sets for use in the fast track will be published with a `mip_era` metadata value of 'CMIP7'.
+This metadata value will appear both in the file's global metadata as well as its metadata on ESGF.
+
+Further details will follow in early 2025.
+
+#### CMIP7
+
+No data available for this phase yet.
+
+This data will be for CMIP7.
+All data sets for use in CMIP7 will be published with a `mip_era` metadata value of 'CMIP7'.
+This metadata value will appear both in the file's global metadata as well as its metadata on ESGF.
+
+Further details will follow after the fast track is underway
+(including details about how updates to this data will be handled over the lifetime of CMIP7).
+
+<!--- end-cmip7-phases-source-ids -->
+
+## Navigating the data
+
+### Recommendation for pre-industrial control
+
+Repeat the 1850 values.
+
+### Grids and frequencies provided
+
+CEDS emissions are provided at monthly resolution, on a 0.5 degree grid, with 50-years per data file as per ESGF formatting conventions. (A 0.1° grid for recent decades is also available.) Emissions are relatively smooth over the early portion of this time period, but annual data is supplied for consistency across the dataset. The files are in netcdf v4 (HDFv5) format with CF-compliant metadata.
+
+Gridded aircraft emissions are also supplied in a separate file with 25 vertical layers (using the CMIP5 historical emissions Lamarque et al. 2010, as drawn from Lee et al., as a template). Note that aircraft emissions are zero in early years, but files are provided for all years consistency.
+
+The gridded emissions incorporate a monthly seasonal cycle by sector drawing largely from the ECLIPSE project, Carbon Tracker, and EDGAR.
+
+VOC speciation is provided at the 23 species resolution used in HTAP and RETRO. 
+
+### Variables provided
+
+Subsequent CEDS data releases since August 2016 are in a format with data in one variable, with sectors included as a dimension. 

```

Comment:
```suggestion
Subsequent CEDS data releases since August 2016 are in a format of one variable per data file, with the sectors included as a dimension of the variable's data. 

```